spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require 'simplecov'
 
-SimpleCov.start
+SimpleCov.start 'rails'
+SimpleCov.minimum_coverage 100
```

Comment:
also need by file:

```suggestion
SimpleCov.minimum_coverage 100
SimpleCov.minimum_coverage_by_file 100
```

Reply:
Is per file not covered by this?
"minimum_coverage 100"

If a file isn't 100% covered, then this won't be either.

Reply:
True